webrtc/RTCPeerConnection-helper.js

```diff
@@ -210,3 +210,37 @@ function test_never_resolve(testFunc, testName) {
     t.step_timeout(t.step_func_done(), 100)
   }, testName);
 }
+
+// Helper function to exchange ice candidates between
+// two local peer connections
+function exchangeIceCandidates(pc1, pc2) {
+  function doExchange(localPc, remotePc) {
```

Comment:
I know. But without proper module system I don't really want to expose the private helper function globally. Putting it as an inner function helps indicate that it is private without going through too much hacks.